bpf/lib/lb.h

```diff
@@ -1188,8 +1196,13 @@ lb4_xlate(struct __ctx_buff *ctx, __be32 *new_daddr, __be32 *new_saddr __maybe_u
 				    BPF_F_PSEUDO_HDR) < 0)
 			return DROP_CSUM_L4;
 	}
+
+	if (!revalidate_data(ctx, &data, &data_end, &ip4))
+		return DROP_INVALID;
```

Comment:
Same remark, would that make sense to pass the pointer down through `lb4_local()`?

Reply:
don't like the idea of pass down the whole header. changed to pass only a bool.